react-native-gutenberg-bridge/android/build.gradle

```diff
@@ -181,6 +181,38 @@ if (buildGutenbergMobileJSBundle) {
     def bundleName = 'index.android.bundle'
     def jsRootDir = '../..'
 
+    // FIXME also check that submodule is properly up to date???
```

Comment:
What do you have in mind here in terms of "up to date"?

Reply:
I was just thinking to confirm that the proper submodule hash was checked out (not behind or ahead). Doing that removes some flexibility since it forces you to always use the submodule ref that is committed, but it also might avoid people seeing weird issues because they forgot to update the submodule after switching branches. If we're going to enforce that the submodule is clean it kind of feels like it makes sense to also verify that the right submodule is checked out, but I didn't implement it yet because I was a bit torn on it.

Reply:
Ah, I see.

Hmm, failing the build if the submodule hash is beind or ahead feels a bit restrictive. I mean, when building code, we understand that if git reports something is dirty (and it will do that if the sunmodule hash is different than the committed one) then that something is what will end up in the build. I trust that the developer will have the same insight for this case too. Especially devs not working on Gutenberg, they will easily notice that the dirty folder is not something they intended to be dirty.

Besides, there's already a hint printed out when there is a mismatch (https://gist.github.com/hypest/e06f6097065728b6db7b7c462f8fef1a/#file-submodulegithash-gradle-L12). We could try to make that look like a warning than just a hint if we're concerned it will not be noticed.